src/installer/pkg/sfx/Microsoft.NETCore.App/monocrossaot.sfxproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="BuildMonoCrossAllTargets">
+  <Sdk Name="Microsoft.DotNet.SharedFramework.Sdk" />
```

Comment:
If you're not going to use any of the built-in targets, then I'd suggest just using the NoTargets SDK.

```suggestion
<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="BuildMonoCrossAllTargets">
```